### DISCUSSION TITLE
Clicking things in chat

### DISCUSSION THREAD

**Discussion by BanksBanksBanks** [ORelio/Minecraft-Console-Client#2046] [Q&A] [answered]
How do I manually click things that are in chat, such as links or embedded commands?
Thanks

**Comment by ORelio** [comment, marked as answer]
MCC has a default setting to show links in chat, if your terminal allows clicking on the link then you can click on it.
https://github.com/MCCTeam/Minecraft-Console-Client/blob/17dbe10ff3c3fc344e7f6bfa82607340eebeb005/MinecraftClient/Resources/config/MinecraftClient.ini#L33

**Reply by BanksBanksBanks** [reply]
I have that setting on, but I don't know if I'm missing something because I'm not given anything extra in the console to click.
![Screenshot (29)](https://user-images.githubusercontent.com/107327232/173665541-6492717d-1aa4-459e-9cae-4f42a05ca89e.png)
![2022-06-14_19 38 16](https://user-images.githubusercontent.com/107327232/173665584-32fd82f9-6533-4c8b-bcc1-c2e116d9764e.png)
Clicking the text on a standard minecraft instance runs the command but I cannot do this or click anything on the console client. How might I go about doing that?

**Reply by ORelio** [reply]
Ah, yes, commands embedded in messages are not handled yet.
